### ISSUE TITLE
Command line parsing of variables breaks for lists variables

### ISSUE THREAD

**Issue by jonathanunderwood** [remind101/stacker#255]
In a blueprint, I have part of the VARIABLES attribute defined as:

``` python
        'SecurityGroups': {
            'type': list,
            'description': 'List of Security Groups for the instance'
        },
```

Calling stacker like this:

```
stacker build -r eu-west-1 -t -var 'SecurityGroups=sg-0d60076a' <env> <conf>
```

Results in a CloudFormation template containing:

```
                "SecurityGroupIds": [
                    "s",
                    "g",
                    "-",
                    "0",
                    "d",
                    "6",
                    "0",
                    "0",
                    "7",
                    "6",
                    "a"
]
```

So, somehow the security group id has become tokenized. It's unclear from the docs/help how -var is supposed to be used, but I don't think this is intended behavior?

**Comment by jonathanunderwood** [comment]
As an extra datapoint, setting the SecurityGroup to a list in the stack definition, results n SecurityGroupIds in the CloudFormation template being an empty list. Something is quite broken here.

This is all with 1.0.0a1

**Comment by phobologic** [comment]
Hey @jonathanunderwood got a partial fix in #266, but a bigger problem is how we pass complex data types in as CLI arguments.  Lists & dicts being the two I'm thinking of now - this could get quite complex for dicts, so lets leave that out of it, and tackle the more immediate issue of lists.

There are two ways I think we could attempt to handle it (though I'm not sure of the difficulty right now):

1. We could allow you to pass the same variable name multiple times, and just append each to the list as you go.

2. We could have a comma-delimited syntax for variables, so if you want multiple values you call `--var` once, but with multiple values separated by a comma.

#2 is probably easiest, though could result in some confusion down the road (we'd also need to provide the ability to escape commas in variables).

@mhahn thoughts?
